src/Features/CustomerEffortScoreTracks.php

```diff
@@ -220,6 +220,17 @@ private function enqueue_to_ces_tracks( $item ) {
 			array()
 		);
 
+		$has_duplicate = array_filter(
+			$queue,
+			function ( $queue_item ) use ( $item ) {
+				return $queue_item['action'] === $item['action']
+				&& $queue_item['label'] === $item['label'];
```

Comment:
Just curious, why did you check the label as well as the action?

Reply:
We might not need to check the label. I was thinking of a situation where you want to display two notifications with a different label, but I might be overthinking :) We can remove the label.